sysadmin_101.rst

```diff
@@ -187,6 +187,15 @@ These backends can be big, and have many considerations behind their design.
   his code will run on. He works closely with his Operations engineers to make sure his
   code is performant and on capacity planning.
 
+  .. _whats-developer-francesca:
+
+* **Francesca is one of five students working on the website for her school's
```

Comment:
Build results are:
> 192: WARNING: Inline strong start-string without end-string.

The `**` at the start of the sentence is meant to bold something, and there's no associated ending `**` which is what is triggering this failure.